src/operator/cudnn_batch_norm.cu

```diff
@@ -154,15 +154,17 @@ class CuDNNBatchNormOp : public Operator {
       out_data[cudnnbatchnorm::kMean].get_with_shape<gpu, 1, real_t>(Shape1(shape_[1]), s);
     Tensor<gpu, 1> save_inv_var =
       out_data[cudnnbatchnorm::kInvVar].get_with_shape<gpu, 1, real_t>(Shape1(shape_[1]), s);
-    float a = 1.0f, b = 0.0f;
+    float a = 1.0f;
+    float b = 0.0f;
+    float b_add = 1.0f;
```

Comment:
We probably need to change it and make it fit into type template, so I guess currently it is fine. Because data and weight will use different beta value, so make two variables may be better.